iceoryx_hoofs/include/iceoryx_hoofs/internal/relocatable_pointer/relative_pointer_data.hpp

```diff
@@ -30,7 +30,7 @@ namespace rp
 class RelativePointerData
 {
   public:
-    using id_t = uint16_t;
+    using identifier_t = uint16_t;
```

Comment:
This is the same `id_t` as in `relative_pointer.hpp` but it should not include this file if possible

Reply:
@elBoberido The unification of the `identifier_t` / `id_t` types and naming will be done on a separate branch once this one and #1521 is merged.